lib/helpers/multiSigInterfaceHelper.ts

```diff
@@ -0,0 +1,151 @@
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+// @ts-ignore -- Throws error because it cant find "../../typechain-types/MultiSig" module. Can't generate typechain-types because of this error -_-
```

Comment:
Weird. But do the typings actually exist?

Reply:
Not until they are generated during compilation. I haven't figured out yet why the test files don't throw the same error, since they also import from typechain.